tests/functional/runahead/default-complex/flow.cylc

```diff
@@ -10,12 +11,12 @@
         # T00, T07, T14, ...
         # oops makes bar spawn as waiting
         PT7H = "foo & oops => bar"
-        # T00, T12, T18... 
+        # T00, T12, T18...
         T00, T12, T18 = "foo"
         # T04...
-        T04 = "run_ok & fail"
+        T04 = "run_ok"
         # T05...
-        T05 = "run_ok_2 & fail"
+        T05 = "run_ok_2"
```

Comment:
It doesn't matter whether these succeed or fail, as `foo.T00` has already failed, so that is what limits the runahead